palasio-design.css

```diff
@@ -1,5 +1,6 @@
 @import url('./components/main-style.css');
 @import url('./components/theme.css');
+@import url('./components/util.css');
```

Comment:
Looks great just check the alert icon in mobile view it was getting little oval shape
Rest everything is good